### DISCUSSION TITLE
win的鼠标移动到linux上后，win的鼠标键盘焦点会丢失，但linux不会有任何反应。

### DISCUSSION THREAD

**Discussion by t298** [debauchee/barrier#1272] [Q&A]
我一直使用win作服务端，archlinux作客户端可以正常使用，但是今天早上archllinux更新后，就出现了这个问题。鼠标移动到linux上后，鼠标键盘都不会有效果。
archlinux,gonme40.3.0,Wayland
win 10 64位。
软件版本是2.3.3.

**Comment by t298** [comment]
好吧，鼠标确实移动到linux上了，可以点击，但是看不见鼠标。键盘还是没用

**Comment by t298** [comment]
当我把linux设置为服务器时，问题被解决了

**Comment by shymega** [comment]
Hi,

Please follow the issue template, and as I, and I suspect the other maintainers/developers, do not speak your language (with all due respect!), could you please report your issue in English?

Thanks.

On Fri, Jul 30, 2021, at 8:05 AM, 298 wrote:
> 
> 
> 当我把linux设置为服务器时，问题被解决了
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/debauchee/barrier/issues/1237#issuecomment-889678430>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAKF2QGAES3DBOJ4AQRNA6DT2JFLFANCNFSM5BHVUTOQ>.
> 


-- 
Kind regards,
--
Dom Rodriguez (also known as shymega)

**Comment by x0xr00t** [comment]
For the easyness of solving this issue, i translated it from original msg to english. 
Maybe this will help for a faster solution and closing the topic. 

{original}
我一直使用win作服务端，archlinux作客户端可以正常使用，但是今天早上archllinux更新后，就出现了这个问题。鼠标移动到linux上后，鼠标键盘都不会有效果archlinux,gonme40.3.0,Wayland
win 10 64位。
软件版本是2.3.3.


{english}
I have been using win as the server, and archlinux as the client can be used normally, but after the update of archllinux this morning, this problem occurred. After the mouse is moved to linux, the mouse and keyboard will have no effect archlinux, gonme40.3.0, Wayland
win 10 64 bit.
The software version is 2.3.3.

**Comment by DhruvaSambrani** [comment]
Barrier does not **yet** support Wayland IIUC.

**Comment by shymega** [comment]
Indeed. Closing.

**Comment by wikicoo** [comment]
It works when using Barrier v2.3.4 as Server.

**Reply by shymega** [reply]
I no longer maintain Barrier. What you are experiencing is a fluke. It's not native support.
